File: implementations/java/src/main/java/io/rcl/core/RCL.java
package io.rcl.core;

import io.rcl.DocumentNode;
import java.util.Map;

public final class RCL {
  private RCL() {}

  public static DocumentNode parse(String text) { return io.rcl.RCL.parse(text); }
  public static Map<String, Object> toObject(String text) { return io.rcl.RCL.toObject(text); }
  public static Map<String, Object> toObject(DocumentNode doc) { return io.rcl.RCL.toObject(doc); }
}

File: implementations/kotlin/src/main/kotlin/io/rcl/core/RCL.kt
package io.rcl.core

import io.rcl.DocumentNode
import io.rcl.Parser
import io.rcl.Converters

object RCL {
  fun parse(text: String): DocumentNode = Parser.parse(text)
  fun toObject(text: String): Map<String, Any?> = toObject(parse(text))
  fun toObject(document: DocumentNode): Map<String, Any?> = Converters.toObject(document)
}
